packages/deprecation-cop/README.md

```diff
@@ -0,0 +1,6 @@
+# Deprecation Cop package
+[![OS X Build Status](https://travis-ci.org/atom/deprecation-cop.svg?branch=master)](https://travis-ci.org/atom/deprecation-cop) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/0s870q5fj3vwihjx/branch/master?svg=true)](https://ci.appveyor.com/project/Atom/deprecation-cop/branch/master) [![Dependency Status](https://david-dm.org/atom/deprecation-cop.svg)](https://david-dm.org/atom/deprecation-cop)
```

Comment:
Recommendation: Remove these status badges, since they point to the statuses for the old atom/deprecation-cop repository.